src/garage/tf/models/gaussian_mlp_model.py

```diff
@@ -74,6 +74,8 @@ class GaussianMLPModel(Model):
                exponential transformation
             - softplus: the std will be computed as log(1+exp(x))
         layer_normalization (bool): Bool for using layer normalization or not.
+        return_distribution (bool): If True, return distribution object.
```

Comment:
this is really nasty. is this temporary?

Reply:
if they really need to be supported in parallel, why not just return everything and use what you need?

Reply:
Yes, this is temporary and the option that returns raw tensors will no longer be needed when regressors accept full trajectories of shape [N, T, *dims]. I don't want to return everything which would break other usage of GaussianMLPModel.

Reply:
I'm sorry, but conditionals controlling returns is so bad I don't want it anywhere in the codebase, even temporarily. I think you need to find another way.

Remember that this code might end up in a release which is supported for 8-12 months.

You can:
* Refactor all the callsites to expect a third return value
* Keep two classes around until you can remove the other variant
* Access the distribution through a property on the network

Reply:
Option 1 doesn't seem that bad:
```sh
rjulian@nyquist:~/code/garage$ rg GaussianMLPModel -l | wc -l
16
```

Reply:
Option 1 sounds good. Will do